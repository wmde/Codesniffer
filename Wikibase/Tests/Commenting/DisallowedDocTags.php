<?php

function noDocComment() {
}

/**
 * These are all fine.
 * @covers
 * @fixme
 * @inheritDoc
 * @license
 * @param
 * @return
 * @throws
 * @var
 * @todo
 * @uses
 */
function validTags() {
}

/**
 * These should all be reported.
 * @cover
 * @inheritdoc
 * @licence
 * @params
 * @property
 * @returns
 * @throw
 * @type
 * @use
 */
function lowerCaseTags() {
}

/**
 * Reporting should be case-insensitive for all-lowercase search patterns.
 * @Cover
 * @Inheritdoc
 * @Licence
 * @Params
 * @Property
 * @Returns
 * @Throw
 * @Type
 * @Use
 */
function upperCaseTags() {
}

/*
 * This is not a PHPDoc comment and should be ignored.
 * @cover
 * @inheritdoc
 * @licence
 * @params
 * @property
 * @returns
 * @throw
 * @type
 * @use
 */
function notADocComment() {
}

/**
 * @package Obsolete
 */
function disallowedTags() {
}
