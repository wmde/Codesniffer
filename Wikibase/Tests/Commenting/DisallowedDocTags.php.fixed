<?php

function noDocComment() {
}

/**
 * These are all fine.
 * @covers
 * @fixme
 * @inheritDoc
 * @license
 * @param
 * @return
 * @throws
 * @var
 * @todo
 * @uses
 */
function validTags() {
}

/**
 * These should all be reported.
 * @covers
 * @inheritDoc
 * @license
 * @param
 * @property
 * @return
 * @throws
 * @var
 * @uses
 */
function lowerCaseTags() {
}

/**
 * Reporting should be case-insensitive for all-lowercase search patterns.
 * @covers
 * @inheritDoc
 * @license
 * @param
 * @Property
 * @return
 * @throws
 * @var
 * @uses
 */
function upperCaseTags() {
}

/*
 * This is not a PHPDoc comment and should be ignored.
 * @cover
 * @inheritdoc
 * @licence
 * @params
 * @property
 * @returns
 * @throw
 * @type
 * @use
 */
function notADocComment() {
}

/**
 * @package Obsolete
 */
function disallowedTags() {
}
