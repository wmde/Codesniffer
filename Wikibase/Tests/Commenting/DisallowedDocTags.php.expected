 24 | ERROR | [x] Found @cover instead of @covers
 25 | ERROR | [x] Found @inheritdoc instead of @inheritDoc
 26 | ERROR | [x] Found @licence instead of @license
 27 | ERROR | [x] Found @params instead of @param
 29 | ERROR | [x] Found @returns instead of @return
 30 | ERROR | [x] Found @throw instead of @throws
 31 | ERROR | [x] Found @type instead of @var
 32 | ERROR | [x] Found @use instead of @uses
 39 | ERROR | [x] Found @Cover instead of @covers
 40 | ERROR | [x] Found @Inheritdoc instead of @inheritDoc
 41 | ERROR | [x] Found @Licence instead of @license
 42 | ERROR | [x] Found @Params instead of @param
 44 | ERROR | [x] Found @Returns instead of @return
 45 | ERROR | [x] Found @Throw instead of @throws
 46 | ERROR | [x] Found @Type instead of @var
 47 | ERROR | [x] Found @Use instead of @uses
 68 | ERROR | [ ] PHPDoc tag @package is not allowed
